zeebe/qa/update-tests/README.md

```diff
@@ -1,9 +1,9 @@
 ## Pre-requisites
 
-To run the upgradability tests, a Docker image for Zeebe must be built with the tag 'current-test'. To do that you can run (in the zeebe-io/zeebe dir):
+To run the upgradability tests, a Docker image for Zeebe must be built with the tag 'current-test'. To do that you can run (in the camunda/zeebe dir):
 
 ```
-docker build --build-arg DISTBALL=dist/target/camunda-zeebe*.tar.gz -t camunda/zeebe:current-test --target app .
+docker build --build-arg DISTBALL='dist/target/camunda-zeebe*.tar.gz' -t camunda/zeebe:current-test --target app .
```

Comment:
👍 This fixes the problem on zsh on macOS.

❓ Does it still work on linux/windows? Might be good to ask some team members to test it before merging.

Reply:
>I usually run docker commands in my Linux subsystem on Windows 🙈😄
So, no idea if this works on Windows 🤷

@tmetzke Still good to try this on your linux subsystem for windows then ;)

Reply:
Works on my Ubuntu subsystem on Windows ✔️ 

```
$ docker build --build-arg DISTBALL='dist/target/camunda-zeebe*.tar.gz' -t camunda/zeebe:current-test --target app .
[+] Building 31.6s (21/21) FINISHED                                                                                                      docker:default
 => [internal] load build definition from Dockerfile                                                                                               0.0s
 => => transferring dockerfile: 7.01kB                                                                                                             0.0s
 => resolve image config for docker.io/docker/dockerfile:1.4                                                                                       0.9s
 => CACHED docker-image://docker.io/docker/dockerfile:1.4@sha256:9ba7531bd80fb0a858632727cf7a112fbfd19b17e94c4e84ced81e24ef1a0dbc                  0.0s
 => [internal] load .dockerignore                                                                                                                  0.0s
 => => transferring context: 308B                                                                                                                  0.0s
 => [internal] load metadata for docker.io/library/ubuntu:jammy@sha256:77906da86b60585ce12215807090eb327e7386c8fafb5402369e421f44eff17e            0.9s
 => [internal] load metadata for docker.io/library/eclipse-temurin:21-jdk-jammy@sha256:7a43bcc2fe4d702c572f39fe200f85d258c081ddc4da4a473ae613b033  1.2s
 => [jre-build 1/2] FROM docker.io/library/eclipse-temurin:21-jdk-jammy@sha256:7a43bcc2fe4d702c572f39fe200f85d258c081ddc4da4a473ae613b033379c9e    0.0s
 => [base 1/3] FROM docker.io/library/ubuntu:jammy@sha256:77906da86b60585ce12215807090eb327e7386c8fafb5402369e421f44eff17e                         0.0s
 => [internal] load build context                                                                                                                 24.2s
 => => transferring context: 204.14MB                                                                                                             24.1s
 => CACHED [base 2/3] RUN --mount=type=cache,target=/var/cache/apt,sharing=locked     --mount=type=cache,target=/var/lib/apt,sharing=locked     -  0.0s
 => CACHED [distball 1/3] WORKDIR /zeebe                                                                                                           0.0s
 => CACHED [jre-build 2/2] RUN jlink      --add-modules ALL-MODULE-PATH      --strip-debug      --no-man-pages      --no-header-files      --comp  0.0s
 => CACHED [java 1/3] COPY --from=jre-build /jre /opt/java/openjdk                                                                                 0.0s
 => CACHED [java 2/3] RUN java -Xshare:dump;                                                                                                       0.0s
 => CACHED [app 1/4] WORKDIR /usr/local/zeebe                                                                                                      0.0s
 => CACHED [app 2/4] RUN groupadd --gid 1001 camunda &&     adduser --system --gid 1001 --uid 1001 --home /usr/local/zeebe camunda &&     chmod g  0.0s
 => CACHED [app 3/4] COPY --link --chown=1001:0 zeebe/docker/utils/startup.sh /usr/local/bin/startup.sh                                            0.0s
 => [distball 2/3] COPY --link dist/target/camunda-zeebe*.tar.gz zeebe.tar.gz                                                                      0.5s
 => [distball 3/3] RUN mkdir camunda-zeebe &&     tar xfvz zeebe.tar.gz --strip 1 -C camunda-zeebe &&     find . -type f -name 'zbctl*' -delete    2.2s
 => [app 4/4] COPY --from=dist --chown=1001:0 /zeebe/camunda-zeebe /usr/local/zeebe                                                                0.5s
 => exporting to image                                                                                                                             1.1s
 => => exporting layers                                                                                                                            1.1s
 => => writing image sha256:0ba9f553d6d31205c0c1960573c1cb913d13cda86dc384f196f452b3e74225dd                                                       0.0s
 => => naming to docker.io/camunda/zeebe:current-test                                                                                              0.0s
```